src/docs/product/accounts/quotas/manage-transaction-quota.mdx

```diff
@@ -138,6 +138,10 @@ If you have a rogue client, Sentry supports blocking an IP from sending data. Na
 
 If you discover a problematic release causing excessive noise, Sentry supports ignoring all events and attachments from that release. Navigate to **[Project] > Settings > Inbound Filters**, then add the release to the "Releases" field.
 
-## 3. SDK Configuration: Tracing Options {#2-sdk-configuration-tracing-options}
+## 3. SDK Filtering: beforeSendTransaction {#1-sdk-filtering-beforesendtransaction}
```

Comment:
Is this heading alias from an ancient version of this doc?
```suggestion
## 3. SDK Filtering: beforeSendTransaction {#1-sdk-filtering-beforesendtransaction}
```

Reply:
I have to assume so? IDK, I just went with it, because fixing it really should be a separate PR if we're going to do it.